docs/_posts/examples/9999-03-01-storage-layer.md

```diff
@@ -0,0 +1,32 @@
+---
+layout:      example
+title:       "Storage Layer"
+categories:  example
+description: "This guide explains how the storage layer works and how to write your own storage adapter"
+---
+
+The storage layer takes care of storing and loading bounded data like user credentials and unbounded data like encryption keys. Note that the storage layer is *not* intended to be your application's database (though may interface with your application's database).
+
+# Usage
+
+Within a spark plugin, the storage layer is available via two properties: `this.boundedStorage` and `this.unboundedStorage`. `boundedStorage` should be used for things with a known size (e.g. user credentials, device registration payloads, etc). `unboundedStorage` should be used for arrays of unknown length (e.g. encryption keys). These properties are bound to the namespace defined by the plugin.
+
+# Adapters
+
+A storage adapter is a proscribed interface to an arbitrary storage backend. Its entry point is `bind()`. Bind accepts one argument, `namespace`, and returns an object containing the methods `del()`, `get()`, and `put()`. These methods are scoped to the `namespace`. For example, the following block would write the value `true` to the key `"proof"` in the `namespace` `"spark-core"`.
```

Comment:
yea, I did that on purpose, but it turns I'm definitely using the wrong word. thanks!